enzy_htp/structure/structure_operation/general.py

```diff
@@ -64,6 +64,24 @@ def remove_non_peptide(stru: Structure) -> Structure:
         ch.delete_from_parent()
     return stru
 
+@dispatch
```

Comment:
You need to add dispatch to the other function too. And please put them as close as possible in the file.

Reply:
Resolved.